crates/bevy_input/src/axis.rs

```diff
@@ -1,3 +1,5 @@
+//! The generic axis type.
```

Comment:
I understand the module docs are not very important but this tells me about as much as the name of the module 😅  What is this axis type used for?

Reply:
I don't see what to put here since items of the modules already have relevant docs.
This docs is only here for being able to add `#![warn(missing_docs)]` on `bevy_input`.